index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Microverse Project</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <div id="main">
+    <h1>hello world</h1>
+    <div id="title">
+      <h1>Mocroverse Project</h1>
+    </div>
+
+    <div id="img-div">
+      <img id="image"
+        src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="">
+        
+      <div id="img-caption">
+        <p> The man who saved a billion lives
+
+        </p>
+        <p id="tribute-info">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase
+          wheat yields - part of his life-long war on hunger. </p>
+      </div>
+    </div>
+    <div>
+      <a id="tribute-link" href="www.google.com" target="_blank">google</a>
+    </div>
+  </div>
+
+
+
+
+
+
+
+
+
+  <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+
+
+
+
+  <script src="js/index.js"></script>
+</body>
```

Comment:
Kindly let all your comments include an explanation, a "why", for each requested change. This will help them see the reasons with you. Some reasons might seem obvious to you but not to the author of the code.